collector/dfsr.go

```diff
@@ -124,8 +124,8 @@ func NewDFSRCollector() (Collector, error) {
 
 		// Connection
 		ConnectionBandwidthSavingsUsingDFSReplicationTotal: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "bandwidth_savings_using_dfs_replication_bytes_total"),
-			"Total amount of bandwidth savings using DFS Replication for this connection, in bytes",
+			prometheus.BuildFQName(Namespace, subsystem, "bandwidth_savings_using_dfs_replication_percentage"),
```

Comment:
I have some vague memory of this being changed *to* bytes from percentages in the original PR? Exposing the bytes if possible would be more useful, IMO (and most frequently, that is what we actually get in the data?)

Reply:
I think you're right: I've reverted the change to these two bandwidth metrics.